Description: Fixes a libcurl4 SSL certificate verify issue.
 TODO: This patch is from the librets SVN and addresses a run-time issue
 which caused connections to fail due to certificate verification failure.
 .
 librets (1.5.3-1realgo) unstable; urgency=low
 .
   * Initial release
Author: Sean Reifschneider <sean+packages@realgo.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: Vendor

--- librets-1.5.3.orig/project/librets/src/CurlEasy.cpp
+++ librets-1.5.3/project/librets/src/CurlEasy.cpp
@@ -209,8 +209,8 @@ void CurlEasy::SetSSLVerify(bool verify)
     if (verify)
         verifies    = 2;
 
-    CurlAssert(curl_easy_setopt(mCurl, CURLOPT_SSL_VERIFYPEER, verify), "Verify SSL Peer");
-    CurlAssert(curl_easy_setopt(mCurl, CURLOPT_SSL_VERIFYHOST, verify), "Verify SSL Host");
+    CurlAssert(curl_easy_setopt(mCurl, CURLOPT_SSL_VERIFYPEER, verifies), "Verify SSL Peer");
+    CurlAssert(curl_easy_setopt(mCurl, CURLOPT_SSL_VERIFYHOST, verifies), "Verify SSL Host");
 }
 
 void CurlEasy::Perform()
